source/_integrations/slack.markdown

```diff
@@ -4,16 +4,20 @@ description: Instructions on how to add Slack notifications to Home Assistant.
 ha_category:
   - Notifications
 ha_release: pre 0.7
+ha_config_flow: true
 ha_domain: slack
 ha_iot_class: Cloud Push
 ha_codeowners:
   - '@bachya'
+  - '@tkdrob'
 ha_platforms:
   - notify
 ---
 
 The `slack` platform allows you to deliver notifications from Home Assistant to [Slack](https://slack.com/).
 
+{% include integrations/config_flow.md %}
```

Comment:
The order is incorrect I think. Let's make a prerequisites chapter first, with the instructions on how to obtain the needed keys first, then add the config flow include after that.